app/src/main/java/com/github/neighbortrader/foodboardapp/requests/GetAllOffersRequest.java

```diff
@@ -0,0 +1,59 @@
+package com.github.neighbortrader.foodboardapp.requests;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.github.neighbortrader.foodboardapp.Constant;
+import com.github.neighbortrader.foodboardapp.clientmodel.Price;
+import com.github.neighbortrader.foodboardapp.clientmodel.Offer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class GetAllOffersRequest <T> extends GetRequest<T> {
+    public final static String TAG = GetAllOffersRequest.class.getSimpleName();
+
+    public GetAllOffersRequest(Context context, OnEventListener callback) {
+        super(context, callback);
+    }
+
+    @Override
+    protected List<T> doInBackground(Void... params) {
```

Comment:
doInBackground ist ein komischer Name. Funktionen Namen sollten eindeutig sein

Reply:
Naja genau das macht sie, der Name ist eindeutig.  `doInBackground` kommt ja nicht von mir sondern von `AsyncTask` welche sie als Methode für eine Hintergrund-Thread implementiert. Sie beschreibt also grundsätzlich genau was sie macht. Zusätzlich kann man  `doInBackground` nicht von außen aufrufen da sich  `AsyncTask` eben um das Asynchrone ausführen kümmert. 

Wenn du willst kann ich die fachliche Logik der Methode noch in eine andere Methode packen mit einen Namen der beschreibt was sie tut.